### ISSUE TITLE
Propagate 'model' property from daydream-controls to laser-controls component?

### ISSUE THREAD

**Issue by evhan55** [aframevr/aframe#2981]
Would it be possible to propagate the `model` property from the `daydream-controls` component up to the `laser-controls` component so that we can hide the controller model but still use laser controls?

https://aframe.io/docs/0.6.0/components/laser-controls.html
https://aframe.io/docs/0.6.0/components/daydream-controls.html

**Comment by donmccurdy** [comment]
Do you still want tracking from daydream controls? If so this should work:

```html
<a-entity laser-controls="hand: left" daydream-controls="model: false"></a-entity>
```

Or just a floating laser that you can use however you want?

```html
<a-entity cursor="fuse: false" raycaster="showLine: true"></a-entity>
```

... but overriding `model:false` for all the different controller types is sort of a pain, so I'd be OK with surfacing a `model` property on `laser-controls` too.

**Comment by evhan55** [comment]
Yes! I would still like Daydream controls, I had already tried this before posting the issue and it didn't seem to work:

```html
<a-entity laser-controls="hand: left" daydream-controls="model: false"></a-entity>
```

I just tried again and it didn't work again, no controller model, but also no tracked laser pointer either.

However, the whole reason I was trying to have Daydream-tracked lasers without a model was because I was trying to reduce lag on loading of the 3D controller model, in trying to resolve a bug where the controller wasn't tracking at all for a colleague of mine.  But, overnight I realized that we had just forgotten to enable the `gamepad-extensions` flag on their Pixel device.

Some follow ups:

- For people who still may want to have controller tracking, laser pointing, but no controller model, the workaround that you posted would be fine (for me, anyway) if it worked as expected, but it doesn't seem to at the moment.  Should I open a new issue for that?
- It might be useful for the A-Frame controller docs to put in reminders that the `webvr` and `gamepad-extensions` flags might need to be set on mobile (and possibly other headsets?)..

Thank you!

**Comment by machenmusik** [comment]
> ... but overriding model:false for all the different controller types manually is sort of a pain

Exactly... If we want to have easy to use laser-controls, we should add values that should cascade downward

**Comment by ngokevin** [comment]
For edge and custom cases, I'd copy and paste the laser-controls component and do what you need rather than adding more API. laser-controls is an out of the box implementation and the code is very simple, it sets other components. If you need to create your own controls, you can modify it or add anothe component that overrides daydream-controls

**Comment by evhan55** [comment]
Thanks, @ngokevin ! Makes sense.

How do you feel about this also working?

```html
<a-entity laser-controls="hand: left" daydream-controls="model: false"></a-entity>
```

**Comment by ngokevin** [comment]
I'll check it out when I can, we should make that case work.

**Comment by machenmusik** [comment]
it may be just changing the setAttribute to 3-argument form so it does not replace existing properties?

**Comment by ngokevin** [comment]
laser-controls does not touch the model property. setAttribute does not replace non-specified properties without the clobber flag.
